conf/machine/iot2050-qemu.conf

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Enes Colpan <enes.colpan@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+require conf/machine/iot2050.conf
+
+DTB_FILES = ""
+
+CONSOLE_KERNEL_PARAMS = ""
+
+PREFERRED_PROVIDER_u-boot-${MACHINE} = "u-boot-qemu-arm64"
+PREFERRED_PROVIDER_u-boot-${MACHINE}-config = "u-boot-qemu-arm64"
```

Comment:
Somehow this generates u-boot-pg1 instead of qemu. Will fix it.

Reply:
Hmm, where should that qemu u-boot come from? A Debian package?

Reply:
It comes from isar-cip-core repository.
https://gitlab.com/cip-project/cip-core/isar-cip-core/-/tree/master/recipes-bsp/u-boot

Reply:
Ah, of course!

Reply:
Fixed.